recipes/dev/fsdp2_recipes.md

```diff
@@ -0,0 +1,12 @@
+## FSDP2 Recipes
+
+This directory contains training recipes for LoRA and QLoRA using FSDP2.
```

Comment:
Want to add brief details here or a link for what FSDP2 is?

Reply:
There's a good doc on torchtitan https://github.com/pytorch/torchtitan/blob/main/docs/fsdp.md